engine/test/integration_tests/e2e_with_selectors/run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -eu
+
+WORK_DIR=$OUT_DIR/$TEST_NAME
```

Comment:
I think we need to change the integration name to `tiflow_with_selectors`.
The old `e2e` seems related to some codes transfered from `e2e` dir.

Reply:
Do you think it might be better if all these integration tests be renamed? After all, `e2e` is a non-descriptive name.

Reply:
Yes. 
We also may remove `e2e` dir in the future. All these tests will move to `integration_test` dir.